### PR TITLE
Added Coursera & Leetcode

### PR DESCRIPTION
**Pull request by diegotf30**

### CODE REVIEW

File: sites.json [hunk 1]
```diff
@@ -3305,6 +3309,16 @@
     },
 
     {
+        "name": "Leetcode",
+        "url": "",
```
Comment (tupaschoal) [inline sites.json]: Is there a link to the info that the account can't be deleted?

File: sites.json [hunk 2]
```diff
@@ -3694,6 +3708,18 @@
     },
 
     {
+        "name": "Mobills",
+        "url": "https://web.mobillsapp.com/Configuracoes/DeletaConta",
+        "difficulty": "easy",
+        "notes": "If you made an accout via Google or Facebook you must first change your password.",
```
Comment (tupaschoal) [inline sites.json]: I'm not sure I get it. First you have to change the password, then it is just a matter of clicking a delete button? 
If that's so, it is not as direct as just clicking, so I would probably put it as medium

**Comment by Nightfirecat** [comment]
Hey @diegotf30, would you be able to answer @tupaschoal's questions? We cannot in good faith merge this PR until they are addressed.

File: sites.json (hunk 1 above)
Comment (diegotf30) [reply sites.json]: I found no articles online, and the support team gave me no response :/

File: sites.json (hunk 2 above)
Comment (diegotf30) [reply sites.json]: The thing is, if you made the account directly on Mobills using an email you can directly delete the account with the button. But if you made the account via Facebook or Google you must take an additional step, so I really don't know how to classify the difficulty

**Comment by diegotf30** [comment]
Hi! Sorry I got caught up on school projects and forgot about this, I have made my comments on the respective changes

File: sites.json (hunk 2 above)
Comment (tupaschoal) [reply sites.json]: I'd probably keep it as easy and rephrase it to something like:
"Just click the delete button. Note that if you have made an account via Google or Facebook, you'll have to change the password first".

File: sites.json (hunk 1 above)
Comment (tupaschoal) [reply sites.json]: Then I'd be fine with just a link to the website.

**Comment by Nightfirecat** [comment]
@diegotf30 Your changes look good, but your branch does not seem to be in a good state to be merged... This is partially due to the fact that your master branch did not have updates from this repo's master branch incorporated before you continued working on it since previous PRs.

I've done the work of reworking your commits into neater commits (one per site, including `gulp` run per commit) [and pushed that branch to nightfirecat/justdelete.me as the "diego-master" branch](https://github.com/Nightfirecat/justdelete.me/tree/diego-master) if you'd like to use it to update your repository. (it seems you have declined to grant access for us to push to your repository)

You should be able to force-update your master branch from this one with the following commands:

```
git remote add nfc git@github.com:nightfirecat/justdelete.me.git  # add my repository as the "nfc" remote
git fetch nfc                                                     # fetch my repository's branches
git branch new-master nfc/diego-master                            # create a new local "new-master" branch from my repository's "diego-master" branch
git checkout new-master                                           # set HEAD to the "new-master" branch
git branch -D master                                              # delete your local copy of "master"
git checkout -b master                                            # create a local "master" branch from HEAD (which is your local "new-master" branch), and move HEAD to that new branch
git branch -D new-master                                          # remove your local "new-master" branch--no longer needed, since local "master" is now a copy of it
git branch -u origin/master                                       # set the new local "master" branch's upstream to your repository's master branch
git push -f                                                       # force-sync your repository's branch to be identical to your local "master" branch
git remote remove nfc                                             # remove my repository from your tracked remotes; it's no longer needed
```

**Comment by diegotf30** [comment]
Weird, I used `git pull` before working the branch and there were no updates, anywho, thanks for the detailed tutorial and help provided! :)

**Comment by Nightfirecat** [comment]
Your master branch's upstream is probably not pointing to this repository; you'd need to do `git fetch && git merge --ff-only jdm/master` (assuming this repository's remote is named `jdm`)
